data/sql_updates/totals_candidate_committee_house_senate.sql

```diff
@@ -0,0 +1,199 @@
+                sum(totals.transfers_from_other_authorized_committee) as transfers_from_other_authorized_committee,
+                sum(totals.transfers_to_other_authorized_committee) as transfers_to_other_authorized_committee,
+                true as full_election
+        from last_totals totals
+        inner join ofec_candidate_election_mv_tmp election on
+            totals.candidate_id = election.candidate_id and
+            totals.cycle <= election.cand_election_year and
+            totals.cycle > election.prev_election_year
+        group by
+            totals.candidate_id,
+            election.cand_election_year
+        ), combined_totals as (
+            select ict.*,
+                totals.last_cash_on_hand_end_period,
+                totals.last_report_type_full,
+                totals.last_debts_owed_to_committee,
+                totals.last_debts_owed_by_committee,
+                totals.last_beginning_image_number,
+                totals.last_report_year
+                --0.0 as cash_on_hand_beginning_of_period
+            from last_totals totals
+            inner join ofec_candidate_election_mv_tmp election on
+                totals.candidate_id = election.candidate_id and
+                totals.cycle = election.cand_election_year
+            inner join intermediate_combined_totals ict on totals.candidate_id = ict.candidate_id and totals.cycle = ict.cycle
+            where totals.cycle > :START_YEAR
+
+        ), cash_period_aggregate as(
+            select
+                p_totals.cash_on_hand_beginning_of_period,
+                election.cand_election_year,
+                p_totals.candidate_id
+            from
+                ofec_candidate_election_mv_tmp election
+                inner join last_totals p_totals on election.candidate_id = p_totals.candidate_id
+                and p_totals.cycle = election.prev_election_year + 2 and p_totals.cycle < election.cand_election_year
+        ), final_combined_total as (
+            select ct.*,
+            cpa.cash_on_hand_beginning_of_period
+            from combined_totals ct
+            inner join cash_period_aggregate cpa on ct.candidate_id = cpa.candidate_id and ct.cycle = cpa.cand_election_year
+        )
+
+        select * from last_totals
+        union all
+        select * from final_combined_total
+;
+
+create unique index on ofec_totals_candidate_committees_house_senate_mv_tmp (candidate_id, cycle, full_election);
+
+create index on ofec_totals_candidate_committees_house_senate_mv_tmp (candidate_id);
+create index on ofec_totals_candidate_committees_house_senate_mv_tmp (election_year);
+create index on ofec_totals_candidate_committees_house_senate_mv_tmp (cycle);
+create index on ofec_totals_candidate_committees_house_senate_mv_tmp (receipts);
+create index on ofec_totals_candidate_committees_house_senate_mv_tmp (disbursements);
```

Comment:
👏 Kudos on figuring out all of the sums and aggregates needed to piece this data together!